install/kubernetes/cilium/templates/etcd-operator/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.etcd.managed .Values.etcd.podDisruptionBudget.enabled }}
```

Comment:
Because this is an existing value, it is the only one enabled by default. I think it would be better if we disabled it by default, like the others.

Reply:
maxUnavailable=1 and podDisruptionBudget.enabled=false have been set by default in the latest commit.